modules/planning/common/history.cc

```diff
@@ -123,6 +123,35 @@ const HistoryObjectDecision* HistoryFrame::GetObjectDecisionsById(
   return &(object_decisions_map_.at(id));
 }
 
+////////////////////////////////////////////////
+// HistoryObjectStatus
+
+void HistoryObjectStatus::Init(
+    const std::string& id, const ObjectStatus& object_status) {
+  id_ = id;
+  object_status_ = object_status;
+}
+
+
+////////////////////////////////////////////////
+// HistoryStatus
+
+void HistoryStatus::SetObjectStatus(
+    const std::string& id, const ObjectStatus& object_status) {
+  object_id_to_status_[id] = object_status;
```

Comment:
Will a DeleteObjectStatus function be necessary for the future to prevent the map from being too large?

Reply:
It will be used in History and as list of tables. If it's longer than a threshold then older ones will be removed.